src/common/utxobased/db/Models/ProcessorTransaction.ts

```diff
@@ -95,9 +97,21 @@ export const toEdgeTransaction = async (
     }
   }
 
+  const confirmations: EdgeTransaction['confirmations'] =
```

Comment:
Using the tx.blockHeight is incorrect. It should be similar to the way it's done in the core:

Where you use the `currentWalletBlockHeight - tx.blockHeight`